conf/messages.cy

```diff
@@ -126,14 +126,10 @@ openPayments.incorrect = Os yw’r hyn sydd arnoch yn anghywir, dylech wirio a a
 openPayments.correctErrors = gywiro gwallau ar eich Ffurflen TAW (yn agor mewn tab newydd)
 openPayments.whatOweHelp.heading = Mae’r hyn sydd arnaf yn anghywir neu ar goll
 openPayments.directDebitAlreadySetUp = Os ydych eisoes wedi trefnu Debyd Uniongyrchol, nid oes angen i chi dalu nawr.
```

Comment:
Same thing as the comment in the English `messages`, just remove the unused DD messages